internal/retry/retry.go

```diff
@@ -80,10 +84,10 @@ func (r *OutboundMiddleware) Call(ctx context.Context, request *transport.Reques
 	rereader, finish := ioutil.NewRereader(request.Body)
 	defer finish()
 	request.Body = rereader
-	boff := policy.backoffStrategy.Backoff()
+	boff := policy.opts.backoffStrategy.Backoff()
```

Comment:
It's all hidden though right? Possible to change this later, not really necessary now though.